src/util/Constants.js

```diff
@@ -81,12 +81,14 @@ exports.DefaultOptions = {
    * @property {string} [api='https://discord.com/api'] Base url of the API
    * @property {string} [cdn='https://cdn.discordapp.com'] Base url of the CDN
    * @property {string} [invite='https://discord.gg'] Base url of invites
+   * @property {string} [template='https://discord.com/template'] Base url of templates
```

Comment:
That is what the client is using, see my comment below.
```suggestion
   * @property {string} [template='https://discord.new'] Base url of templates
```